python/cudf/cudf/tests/test_orc.py

```diff
@@ -310,7 +310,7 @@ def test_orc_read_skiprows(tmpdir):
     writer = pyorc.Writer(buff, pyorc.Struct(a=pyorc.Boolean()))
     tuples = list(
         map(
-            lambda x: (None,) if x[0] is pd.NA else x,
+            lambda x: (None,) if x[0] is pd.NA else (bool(x[0]),),
```

Comment:
@hyperbolic2346 It seemed during our discussion that we could avoid defining `df` entirely here. Something like:

```python
def test_orc_read_skiprows(tmpdir):
    buff = BytesIO()
    data = [
        True,
        False,
        True,
        False,
        None,
        True,
        True,
        True,
        False,
        None,
        False,
        False,
        True,
        True,
        True,
        True,
    ]
    writer = pyorc.Writer(buff, pyorc.Struct(a=pyorc.Boolean()))
    writer.writerows([(d,) for d in data])
    writer.close()

    skiprows = 10

    expected = cudf.read_orc(buff)[skiprows::].reset_index(drop=True)
    got = cudf.read_orc(buff, skiprows=skiprows)

    assert_eq(expected, got)
```

Was there a reason you decided against this?

Reply:
I'm happy to change this test as we discussed, but I didn't think it was something to sneak in under the guise of "fixing deprecation warnings". We can add this here or I can make another PR to simplify the test.

Reply:
Okay, that's reasonable given the name of the PR. I would approve that change in this PR if you think it is appropriate, or we can do it in another PR. I do think the simplification would be helpful for clarity.

Reply:
Feel free to resolve this whichever way you prefer. I approved the PR. Thanks!